php/cli/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM quay.io/centos/centos:stream9
 ARG PHP_VERSION
 ARG PHP_EXTENSIONS="bcmath cli common gd gmp intl json mbstring \
     mcrypt sodium mysqlnd pgsql opcache pdo pdo_pgsql pecl-msgpack pecl-amqp pecl-redis pecl-imagick pecl-zip pecl-openswoole process soap xml xmlrpc"
+ARG COMPOSER_VERSION=2.7
```

Comment:
@lfluvisotto Rather than locking us into version 2.7, I think it would be better if we use a more generic tag like `latest` or simply the major version number `2`. Right now the "latest" tag and the "2" tag in Docker Hub refer to the same image hash.

Reply:
@bap14, good catch, it went unnoticed by me, changed, please, check again.

![image](https://github.com/user-attachments/assets/469e53b7-101b-4b29-84c7-58540805d0fb)